### PR TITLE
[kubernetes.core] add molecule jobs for ansible 2.9 and 2.10

### PR DESCRIPTION
**Pull request by abikouo**
add molecule jobs for ansible stable-2.9, stable-2.10 and stable-2.11

### CODE REVIEW

File: zuul.d/nodesets.yaml [hunk 1]
```diff
@@ -951,3 +951,18 @@
         nodes:
           - Trendmicro-DeepSecurity-20.0.344
           - ubuntu-bionic
+
+- nodeset:
+    name: kind-centos
+    nodes:
+      - name: master
+        label: fedora-34-1vcpu
```
Comment (pabelanger) [inline zuul.d/nodesets.yaml:959]: okay, centos-8-stream should be good to move back too

**Comment by gravesm** [comment]
@abikouo If at all possible, I'd prefer to get https://github.com/ansible-collections/kubernetes.core/pull/198 merged before we remove the integration tests. As it stands, this PR is removing test coverage.

**Comment by abikouo** [comment]
> @abikouo If at all possible, I'd prefer to get [ansible-collections/kubernetes.core#198](https://github.com/ansible-collections/kubernetes.core/pull/198) merged before we remove the integration tests. As it stands, this PR is removing test coverage.

yes indeed, this is just for test purpose.
Your pull request will be merged before this one.

**Comment by abikouo** [comment]
replaced with #1080

**Comment by goneri** [comment]
> replaced with #1080

#1080 is an issue, not a PR.

**Comment by abikouo** [comment]
edit #1081

**Pull request closed without merging**